### PR TITLE
Implement new color suggestion

### PR DESCRIPTION
**Pull request by joshmoore**
see: https://github.com/ome/omero-ms-zarr/pull/71

### CODE REVIEW

**Comment by joshmoore** [comment]
Fixing the build will require #47 to be merged.

File: ome_zarr/reader.py [hunk 1]
```diff
@@ -182,13 +181,14 @@ class Label(Spec):
     @staticmethod
     def matches(zarr: BaseZarrLocation) -> bool:
         """If label-specific metadata is present, then return true."""
-        # FIXME: this should be the "label" metadata soon
-        return bool("colors" in zarr.root_attrs or "image" in zarr.root_attrs)
+        return bool("image-label" in zarr.root_attrs)
```
Comment (sbesson) [inline ome_zarr/reader.py:184]: Should the `Spec` be renamed to `ImageLabel`?

**Comment by joshmoore** [comment]
Re-opening to trigger a rebuild with #47

File: ome_zarr/reader.py (hunk 1 above)
Comment (joshmoore) [reply ome_zarr/reader.py:180]: Interesting. No immediate objections, but to some extent I think we're "safer" in ome-zarr-py in the sense that it's clearly our namespace whereas in the json someone else could propose a "label" key.

**Comment by joshmoore** [comment]
_grrr_... even the repeated clones are now failing.

**Comment by joshmoore** [comment]
_whew_ Green on a manual re-run of the Windows workflow.

**Comment by joshmoore** [comment]
Merging along with ome/omero-ms-zarr#71 in order to enable a basic workflow.

**Pull request merged**